### ISSUE TITLE
weird results after conversion to Tensorflow

### ISSUE THREAD

**Issue by fbarilla** [fizyr/keras-retinanet#1227]
After applying some tooling to convert the Keras/Retinanet model (.h5) into a Tensorflow model (.pb), I'm facing some issues. When I run the Keras/Retinanet model against an image:
	boxes, scores, labels = model.predict_on_batch(np.expand_dims(image, axis=0))
the boxes, scores and labels are populated with the expected results. Just for information, the model gives me back the 9 detected objects I was expecting. 

For the Tensorflow/Retinanet model, the situation is quite different.   
	regression, classification = sess.run([tf_classification, tf_regression], feed_dict={tf_input: image})

	filter_detections_layer = keras_retinanet.layers.FilterDetections(nms=True)
	boxes, scores, labels = filter_detections_layer.call([regression, classification])

Both 'regression' and 'classification' have the right format:

	Tensor("classification/concat:0", shape=(?, ?, 3), dtype=float32)
	Tensor("regression/concat:0", shape=(?, ?, 4), dtype=float32)

However, after applying the 'FilterDetections', the boxes, scores and labels are populated with ~240 objects and the bounding boxes have a weird format 

	[[ 0.00330357 -0.50109524 -0.46569705  0.5642642 ]
	 [ 0.00824136 -1.2870306  -0.4677258  -0.2555487 ]
	 [ 0.5877216  -1.5296552   0.06298425 -0.5602327 ]
	 ...

For the same image with the Keras/Retinanet model, bounding boxes are:

	[[827.52124 229.86569 880.2724  363.2836 ]
	 [ 19.23209 294.3282  102.16435 346.58243]
	 [550.0928  279.2105  586.58453 318.4497 ]
	 ...

Looks like I've been missing something. I've been digging into the keras-retinanet code for a couple of days but so far, I'm stuck. Any idea what's wrong in my logic or code ?
